Detectors/PHOS/simulation/src/RawWriter.cxx

```diff
@@ -244,12 +240,16 @@ void RawWriter::createRawBunches(const std::vector<o2::phos::Digit*>& channelDig
       }
     }
   }
+  if (currentBunch.mADCs.size()) {
+    bunchHG.push_back(currentBunch);
+    currentBunch.mADCs.clear();
+  }
   if (isLGFilled) {
     bunchLG.clear();
     currentBunch.mADCs.clear();
     for (int i = kNPHOSSAMPLES - 1; i--;) {
```

Comment:
are you sure you want to skip processing of  ``sample[kNPHOSSAMPLES - 1]``, I see you fixed similar loop above.

Reply:
Thanks, fixed